src/Data/String.purs

```diff
@@ -37,7 +40,8 @@ module Data.String
   ) where
 
 import Prelude
-
+import Data.Array as A
+import Data.Functor as F
```

Comment:
Without a qualified Functor import, I receive the error:
`map f = toCharArray >>> map f >>> fromCharArray`
```
Warning found:
in module Data.String
at src/Data/String.purs line 134, column 25 - line 134, column 28

  Shadowed definitions are in scope for value map from the following open imports:

    import Prelude

  These will be ignored and the local declaration will be used.


See https://github.com/purescript/documentation/blob/master/errors/ScopeShadowing.md for more information,
or to contribute content related to this warning.
```